great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -65,6 +65,15 @@
 logger = logging.getLogger(__name__)
 
 
+def _extract_fluent_datasources(config_dict: dict) -> dict:
+    datasources = config_dict.get("datasources", {})
+    fds_names: list[str] = []
+    for ds_name, ds in datasources.items():
+        if "type" in ds:
+            fds_names.append(ds_name)
+    return {name: datasources.pop(name) for name in fds_names}
```

Comment:
Not 100% necessary but a docstring to explain why we're pulling these out might be helpful for other devs

Reply:
done